### ISSUE TITLE
Show React version

### ISSUE THREAD

**Issue by taneliang** [MLH-Fellowship/scheduling-profiler-prototype#73]
From Brian's 17 July email and [doc](https://docs.google.com/document/d/1_UKje2GuvQhstXWeVsQwHMJ3qc80-bMaTrTMfAP5c4o/edit#):

> Show lane labels and React version in the UI somewhere

- Lane labels: we may want to wait until #50 is done in case we want to use WebGL. (**Update**: #80 implemented a view system and is merged; no longer blocking)
- React version: blocked by https://github.com/MLH-Fellowship/react/issues/35 (**Update**: no longer blocking; this was implemented in https://github.com/facebook/react/pull/19553)

**Comment by kartikcho** [comment]
Hey @bvaughn, what do you think about keeping lane labels in the tooltip (as it is right now) instead of drawing labels at the start of a lane?

It will take up some valuable space on the left when we port it to Devtools. Adding labels might also make the Devtools UI look too 'packed' in my opinion.

I think the current tooltip does a good job of showing lane information. If we still want to put emphasis on the lane info, we can try moving it to the top of the tooltip or make the font bold.

Screenshot of the current tooltip:
![image](https://user-images.githubusercontent.com/48270786/90121508-ae1ae180-dd79-11ea-9b54-f90eaf9a78ad.png)

**Comment by bvaughn** [comment]
Good call. It's probably fine to keep the labels in the tooltip for now– particularly since we don't have anything meaningful to show for them (other than the number). :smile:
